.azure-pipelines/Windows-CI.yml

```diff
@@ -48,17 +48,34 @@ jobs:
       $mlir_match = Select-String -Path onnx-mlir\utils\clone-mlir.sh -Pattern "git checkout ([\da-f]+)"
       $mlir_tag = "MLIR_Windows_$($mlir_match.Matches.Groups[1].Value)"
 
-      $build_list = az pipelines build list --org https://dev.azure.com/onnx-pipelines --project onnx --definition-ids 9 --tags $mlir_tag --status completed --result succeeded | ConvertFrom-Json
-      $most_recent = $build_list | Sort-Object startTime | Select-Object -Last 1
-      $build_id = $most_recent.id
+      # Use this version to select artifact by tag, once permissions are enabled this will be significantly faster
```

Comment:
Which parts would have to be removed to make this section work? Everything beside it?

Reply:
That's a good point, I'll clarify in the comment